pkg/combustion/registry.go

```diff
@@ -173,6 +222,19 @@ func createRegistryCommand(ctx *image.Context, commandName string, args []string
 	return cmd, logFile, nil
 }
 
-func IsEmbeddedArtifactRegistryEmpty(registry image.EmbeddedArtifactRegistry) bool {
-	return len(registry.HelmCharts) == 0 && len(registry.ContainerImages) == 0
+func IsEmbeddedArtifactRegistryConfigured(ctx *image.Context) bool {
+	return len(ctx.ImageDefinition.Kubernetes.HelmCharts) != 0 ||
+		len(ctx.ImageDefinition.EmbeddedArtifactRegistry.ContainerImages) != 0 ||
+		len(ctx.ImageDefinition.Kubernetes.Manifests.URLs) != 0
+}
+
+func localManifestsConfigured(localManifestSrcDir string) (bool, error) {
```

Comment:
Can we reuse `isComponentConfigured` instead?

Reply:
Yes, I forgot about that